rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,39 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+linter for the programming language the repo is written in, where one is
+available.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked this
+can lead to inconsistencies in style, which over time can make code messy, and
+difficult to maintain and review. It can also lead to bikeshedding over code
+style in code reviews, which can cause conflict and slow down code review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+standard linting tool to make sure code meets a standard style. Examples of this
+include (but are not limited to):
+
+- [Standard.rb](https://github.com/testdouble/standard)
+- [ESLint](https://eslint.org/)
+- [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer)
+- [Black](https://github.com/psf/black)
+
+When a linter is installed, each package MUST run a check using the linting tool
+as part of the default CI build (preferably before the main test suite runs). We
```

Comment:
I disagree here. I think the linter should run *after* the test suite. It's incredibly frustrating to be looking for test failures and have something fail on the linter, which you then have to fix before finding out that your code is wrong in more fundamental ways. I prefer to run the linter after the tests have passed, including on CI.